src/components/transformation/create/index.tsx

```diff
@@ -148,26 +147,40 @@ function TransformationCreate() {
                 );
             }
             const draftId: string = draft.data[0].id;
-            const specsByName = await getLiveSpecsByCatalogNames(
-                null,
-                Array.from(selectedCollectionSet)
+            const spec = {
+                key: ['/your_key'],
+                schema: {
+                    properties: {
+                        your_key: {
+                            type: 'string',
+                        },
+                    },
+                    type: 'object',
+                    required: ['your_key'],
+                },
+                derive: {
+                    using: {
+                        sqlite: { migrations: ['migration_1.sql'] },
```

Comment:
```
{
    "schema": {
      "type": "object",
      "properties": {
        "your_key": {
          "type": "string"
        }
      },
      "required": [
        "your_key"
      ]
    },
    "key": [
      "/your_key"
    ],
    "derive": {
      "using": {
        "sqlite": {
          "migrations": [
            "BASE.migration.0.sql"
          ]
        }
      },
      "transforms": [
        {
          "name": "TRANSFORM",
          "source": "demo/wikipedia/recentchange",
          "lambda": "BASE.lambda.TRANSFORM.sql"
        }
      ]
    }
}
```

This is a canonical SQLite template to generate ☝️ 
This is a canonical TypeScript template to generate 👇 

```
{
    "schema": {
      "type": "object",
      "properties": {
        "your_key": {
          "type": "string"
        }
      },
      "required": [
        "your_key"
      ]
    },
    "key": [
      "/your_key"
    ],
    "derive": {
      "using": {
        "typescript": {
          "module": "BASE.ts"
        }
      },
      "transforms": [
        {
          "name": "TRANSFORM",
          "source": "demo/wikipedia/recentchange"
        }
      ]
    }
  }
```

@jshearer TRANSFORM and BASE are templated parameters for the base name (ex, BASE of `johnny/testing` is `testing`) and the transform name respectively. Using these names exactly is desired because it means that, if the spec is authored, published, and then pulled again, the filenames will stay consistent.

I tested these by creating a draft and manually updating it via:

```
# Inspect raw draft specs
$ RUST_LOG=info ~/flowctl raw get --table draft_specs --query draft_id=eq.09:68:5a:e4:74:60:08:00 | jq '.'

# Update draft spec
$ RUST_LOG=info ~/flowctl raw update --table draft_specs --query id=eq.09:68:73:65:9e:60:50:00 --body "$(<path/to/row.json)"
```

Note that in row.json above, you need to wrap everything in the table column names `{"catalog_name": "johnny/testing", "spec": { ... }}` etc.